assets/dev/js/editor/document/elements/commands/reset-settings.js

```diff
@@ -30,6 +30,8 @@ export class ResetSettings extends $e.modules.editor.document.CommandHistoryBase
 				defaultValues[ controlName ] = control.default;
 			} );
 
+			defaultValues.__globals__ = {};
```

Comment:
unrelated to this PR, but does setting the default value to the control mean we're saving it to the database rather than just resetting it?
shouldn't it be the same solution as what you've done with the globals? (basically removing the data and letting the system set the defaults)